spec/features/address/six_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
```

Comment:
Avoid naming these files `one_spec`, `two_spec` since that doesn't communicate anything about what this is testing.